estimator/lwe_bkw.py

```diff
@@ -72,7 +72,7 @@ def t1(params: LWEParameters, ell, t2, b, ntest=None):
             ntest = CodedBKW.ntest(params.n, ell, t1, t2, b, params.q)
         sigma_set = sqrt(params.q ** (2 * (1 - ell / ntest)) / 12)
         Ni = [CodedBKW.N(i, sigma_set, b, params.q) for i in range(1, t2 + 1)]
-        t1 = len([e for e in Ni if e <= b])
+        t1 = sum(e <= b for e in Ni)
```

Comment:
This is a rare case of me thinking the old version was more readable.

Reply:
@malb , I understand what you mean. Personally, I think they are both not ideal :D. However, I would champion the change purely because it does not require creating another list. Bools are guaranteed to work as expected here as their instances inherit from numbers.Integral.

As a compromise I have resorted to adding a clarifying comment of what the line of code is doing; otherwise, quite happy to revert the change.